<!doctype html>
<!--
  Minimal Mistakes Jekyll Theme 4.24.0 by Michael Rose
  Copyright 2013-2020 Michael Rose - mademistakes.com | @mmistakes
  Free for personal and commercial use under the MIT license
  https://github.com/mmistakes/minimal-mistakes/blob/master/LICENSE
-->
<html lang="ja" class="no-js">
  <head>
    <meta charset="utf-8">

<!-- begin _includes/seo.html --><title>柳澤 渓甫 | Keisuke Yanagisawa</title>
<meta name="description" content="An assistant professor in Science Tokyo.">


  <meta name="author" content="Keisuke Yanagisawa">
  


<meta property="og:type" content="website">
<meta property="og:locale" content="ja_JP">
<meta property="og:site_name" content="柳澤 渓甫 | Keisuke Yanagisawa">
<meta property="og:title" content="            柳澤 渓甫      Keisuke Yanagisawa      ">
<meta property="og:url" content="https://keisuke-yanagisawa.github.io/">


  <meta property="og:description" content="An assistant professor in Science Tokyo.">











  

  


<link rel="canonical" href="https://keisuke-yanagisawa.github.io/">




<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "Person",
      "name": "Keisuke Yanagisawa",
      "url": "https://keisuke-yanagisawa.github.io/"
    
  }
</script>







<!-- end _includes/seo.html -->



  <link href="/feed.xml" type="application/atom+xml" rel="alternate" title="柳澤 渓甫 | Keisuke Yanagisawa Feed">


<!-- https://t.co/dKP3o1e -->
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="/assets/css/main.css">
<link rel="stylesheet" href="/assets/css/user_define.css">
<link rel="preload" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
<link rel="preload" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
<noscript><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"></noscript>
<noscript><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css"/></noscript>

<!-- for MathJax -->
<script async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js" id="MathJax-script"></script>



    <!-- start custom head snippets -->

<!-- insert favicons. use https://realfavicongenerator.net/ -->

<!-- end custom head snippets -->

  </head>

  <body class="layout--single">
    <nav class="skip-links">
  <ul>
    <li><a href="#site-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
    <li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
    <li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
  </ul>
</nav>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->

    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        
        <a class="site-title" href="https://keisuke-yanagisawa.github.io/en">
          柳澤 渓甫 | Keisuke Yanagisawa
          
        </a>
        <ul class="visible-links">
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/profile">Profile</a>
                </li>
              
            
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/research">Research</a>
                </li>
              
            
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/publications">Publications</a>
                </li>
              
            
              
                <li class="masthead__menu-item" style="color: gray"> | </li>
              
            
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/..">  JP</a>
                </li>
              
            </ul>
        
        <button class="search__toggle" type="button">
          <span class="visually-hidden">Toggle search</span>
          <i class="fas fa-search"></i>
        </button>
        
        <button class="greedy-nav__toggle hidden" type="button">
          <span class="visually-hidden">Toggle menu</span>
          <div class="navicon"></div>
        </button>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>


    <div class="initial-content">
      









<div id="main" role="main">
  
  <div class="sidebar sticky">
  


<div itemscope itemtype="https://schema.org/Person" class="h-card">

  
    <div class="author__avatar">
      <a href="https://keisuke-yanagisawa.github.io/">
        <img src="/assets/img/profile.png" alt="Keisuke Yanagisawa" itemprop="image" class="u-photo">
      </a>
    </div>
  

  <div class="author__content">
    <h3 class="author__name p-name" itemprop="name">
      <a class="u-url" rel="me" href="https://keisuke-yanagisawa.github.io/" itemprop="url">Keisuke Yanagisawa</a>
    </h3>
    <div class="author__bio p-note" itemprop="description">
      <p>
        Assistant Professor, Institute of Science Tokyo
      </p>
    </div>
  </div>

  <div class="author__urls-wrapper">
    <ul class="author__urls social-icons">
      
        <li itemprop="homeLocation" itemscope itemtype="https://schema.org/Place">
            <i class="fas fa-fw fa-map-marker-alt" aria-hidden="true"></i> <span itemprop="name" class="p-locality">Tokyo, Japan</span>
        </li>
        <br>
      
      
      
        
          
            <li><a href="/assets/CurriculumVitae.pdf" rel="nofollow noopener noreferrer me"><i class="fas fa-fw fa-file-pdf" aria-hidden="true"></i><span class="label">Curriculum Vitae</span></a></li>
          
        
          
            <li><a href="https://orcid.org/0000-0003-0224-0035" rel="nofollow noopener noreferrer me"><i class="fab fa-fw fa-orcid" aria-hidden="true"></i><span class="label">ORCID</span></a></li>
          
        
          
            <li><a href="https://www.researchgate.net/profile/Keisuke-Yanagisawa" rel="nofollow noopener noreferrer me"><i class="fab fa-fw fa-researchgate" aria-hidden="true"></i><span class="label">ResearchGate</span></a></li>
          
        
          
            <li><a href="https://scholar.google.co.jp/citations?user=d-5VVzAAAAAJ" rel="nofollow noopener noreferrer me"><i class="ai fa-fw ai-google-scholar-square" aria-hidden="true"></i><span class="label">Google Scholar</span></a></li>
          
        
          
            <li><a href="http://github.com/keisuke-yanagisawa" rel="nofollow noopener noreferrer me"><i class="fab fa-fw fa-github" aria-hidden="true"></i><span class="label">GitHub</span></a></li>
          
        
          
            <li><a href="https://speakerdeck.com/keisukeyanagisawa" rel="nofollow noopener noreferrer me"><i class="fa-brands fa-speaker-deck" aria-hidden="true"></i><span class="label">Speaker Deck</span></a></li>
          
        
          
            <li><a href="mailto:yanagisawa@c.titech.ac.jp" rel="nofollow noopener noreferrer me"><i class="fas fa-fw fa-envelope-square" aria-hidden="true"></i><span class="label">Email</span></a></li>
          
        
      

      

      <!--
  <li>
    <a href="http://link-to-whatever-social-network.com/user/" itemprop="sameAs" rel="nofollow noopener noreferrer me">
      <i class="fas fa-fw" aria-hidden="true"></i> Custom Social Profile Link
    </a>
  </li>
-->
    </ul>
  </div>
</div>
  
  </div>



  <article class="page h-entry" itemscope itemtype="https://schema.org/CreativeWork">
    
    
    
    

    <div class="page__inner-wrap">
      
        <header>
          
          


        </header>
      

      <section class="page__content e-content" itemprop="text">
        
          <aside class="sidebar__right ">
            
              <nav class="toc">
                <header><h4 class="nav__title"><i class="fas fa-file-alt"></i> On this page</h4></header>
                <ul class="toc__menu"><li><a href="#about-me">About me</a></li><li><a href="#recent-news">Recent news</a><ul><li><a href="#2025-january-june">2025 January-June</a></li><li><a href="#2024-july-december">2024 July-December</a></li><li><a href="#2024-january-june">2024 January-June</a></li><li><a href="#2023-july-december">2023 July-December</a></li><li><a href="#2023-january-june">2023 January-June</a></li><li><a href="#2022-july-december">2022 July-December</a></li><li><a href="#2022-january-june">2022 January-June</a></li><li><a href="#news-archive">News archive</a></li></ul></li><li><a href="#tools">Tools</a><ul><li><a href="#exprorer-2021">EXPRORER (2021)</a></li><li><a href="#spresso-2017">Spresso (2017)</a></li></ul></li></ul>

              </nav>
            
            
            
          </aside>
        
        <h1 id="about-me">About me</h1>
<p>I’m an assistant professor in Institute of Science Tokyo. I have proposed several methods with computational, algorithmical, and physicochemical consideration.</p>

<p><br /></p>

<h1 id="recent-news">Recent news</h1>
<h2 id="2025-january-june">2025 January-June</h2>
<ul>
  <li>2025/06 Oral presentations will be given at IPSJ SIG Bioinformatics and Genomics (SIGBIO).
    <ul>
      <li>Ryoya Nakano, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Application of the Ising model to fragment-based protein-ligand docking”, <em>IPSJ SIG Technical Report</em>, <strong>2025-BIO-82</strong>(46): 1-8, 2025/06/22.</li>
      <li>Masahiro Shimizu, Masatake Sugita, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Development of a replica parameter optimization method for REUS MD and its application to membrane permeability prediction of cyclic peptides”, <em>IPSJ SIG Technical Report</em>, <strong>2025-BIO-82</strong>(45): 1-8, 2025/06/22.</li>
      <li>Satoshi Yoneyama, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Selection of Representative Fragment Sets for Fragment-Based Virtual Screening Focusing on 3D Structural Similarity”, <em>IPSJ SIG Technical Report</em>, <strong>2025-BIO-82</strong>(44): 1-8, 2025/06/22.</li>
      <li>Kaho Akaki, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Probe atom distributions obtained from mixed-solvent molecular dynamics improves scoring of docking calculations”, <em>IPSJ SIG Technical Report</em>, <strong>2025-BIO-82</strong>(43): 1-8, 2025/06/22.</li>
    </ul>
  </li>
  <li>2025/06 Poster presentations will be given at the 25th Annual Meeting of the Protein Science Society of Japan.
    <ul>
      <li>Kaho Akaki, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Probe atoms distribution in mixed-solvent molecular dynamics improves scoring of docking calculation”, <em>The 25th Annual Meeting of the Protein Science Society of Japan</em>, 1P-061, 2025/06/18-20.</li>
      <li>Chie Motono, <strong>Keisuke Yanagisawa</strong>, Jun Koseki, Kenichiro Imai. “CrypTothML: A Combined Mixed-Solvent Molecular Dynamics Simulation and Machine Learning Method for Cryptic Site Prediction”, <em>The 25th Annual Meeting of the Protein Science Society of Japan</em>, 3P-038, 2025/06/18-20.</li>
      <li>Masahiro Shimizu, Masatake Sugita, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Development of an Optimization Method for REUS MD Parameters and Its Application to the Prediction of Cyclic Peptide Membrane Permeability”, <em>The 25th Annual Meeting of the Protein Science Society of Japan</em>, 1P-064, 2025/06/18-20.</li>
      <li>Kei Terakura, Masatake Sugita, Takuya Fujie, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Kinetic Analysis of Membrane Permeation Process of Cyclic Peptides Using Markov State Models with Molecular Dynamics Simulations”, <em>The 25th Annual Meeting of the Protein Science Society of Japan</em>, 2P-044, 2025/06/18-20.</li>
      <li>Masatake Sugita, Yudai Noso, Jianan Li, Takuya Fujie, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Protocol for Membrane Permeability Prediction of Cyclic Peptides by Combining Molecular Dynamics Simulations and Machine Learning”, <em>The 25th Annual Meeting of the Protein Science Society of Japan</em>, 3P-040, 2025/06/18-20.</li>
    </ul>
  </li>
  <li>2025/03 An oral presentation will be given at IPSJ SIG Bioinformatics and Genomics (SIGBIO).
    <ul>
      <li>Tomoya Saito, Masayoshi Shimizu, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “フラグメントの類似性を考慮した化合物立体配座検索システムの構築”, <em>IPSJ SIG Technical Report</em>, <strong>2025-BIO-81</strong>(12): 1-8, 2025/03/07.</li>
    </ul>
  </li>
</ul>

<h2 id="2024-july-december">2024 July-December</h2>
<ul>
  <li>2024/12 A poster presentation will be given at the 52nd SAR symposium.
    <ul>
      <li>Genki Kudo, <strong>Keisuke Yanagisawa</strong>, Ryunosuke Yoshino, Takatsugu Hirokawa. “3D Protein-Protein Interaction Surface Profile using Mixed-Solvent Molecular Dynamics”, <em>第52回構造活性相関シンポジウム</em>, KP20, 2024/12/12-13.</li>
    </ul>
  </li>
  <li>2024/10 I got a funding from Kayamori Foundation of Informational Science Advancement.
    <ul>
      <li>“共溶媒分子動力学法によるタンパク質化合物ドッキング計算のスコア関数の改善”, Research Funding Program, Kayamori Foundation of Informational Science Advancement, PI, 1 000 000 yen, 2024/10-2026/09.</li>
    </ul>
  </li>
  <li>2024/10 Poster presentations will be given at Chem-Bio Informatics Society(CBI) Annual Meeting 2024.
    <ul>
      <li>Tomoya Saito, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Development of an efficient compound 3D conformer search system based on relative position of fragments”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2024</em>, P07-33, 2024/10/28-31.</li>
      <li>Masayoshi Shimizu, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Development of a compound pre-screening method based on docking of fragments”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2024</em>, P07-16, 2024/10/28-31.</li>
      <li>Kaho Akaki, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Acquisition of Bias Information for Protein-Ligand Docking by Mixed-Solvent Molecular Dynamics”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2024</em>, P07-15, 2024/10/28-31.
        <ul>
          <li>Winner of the Like! Poster Award (Selected by Conference Attendee Votes)</li>
        </ul>
      </li>
      <li><strong>Keisuke Yanagisawa</strong>, Takuya Fujie, Kazuki Takabatake, Yutaka Akiyama. “QUBO Problem Formulation of Fragment-Based Protein–Compound Flexible Docking”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2024</em>, P07-14, 2024/10/28-31.
        <ul>
          <li><a href="https://speakerdeck.com/keisukeyanagisawa/quadratic-unconstrained-binary-optimization-qubo-problem-formulation-of-fragment-based-protein-compound-flexible-docking">Presentation poster</a></li>
        </ul>
      </li>
      <li>Masatake Sugita, Yudai Noso, Takuya Fujie, Jianan Li, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Development of Prediction Models for Membrane Permeability of Cyclic Peptides using 3D Descriptors obtained from Molecular Dynamics Simulations and 2D Descriptors”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2024</em>, P07-05, 2024/10/28-31.</li>
      <li>Jun Koseki, Chie Motono, <strong>Keisuke Yanagisawa</strong>, Ryunosuke Yoshino, Takatsugu Hirokawa, Kenichiro Imai. “Development of the Cryptic Site searching method with Mixed-solvent molecular dynamics and Topological data analyses methods”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2024</em>, P04-03, 2024/10/28-31.</li>
      <li>Jianan Li, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “CycPeptMP: Development of Membrane Permeability Prediction Model of Cyclic Peptides with Multi-Level Molecular Features and Data Augmentation”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2024</em>, P03-11, 2024/10/28-31.</li>
      <li>Kei Terakura, Masatake Sugita, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Kinetic Analysis of Membrane Permeation Process of Cyclic Peptides Using Markov State Models with Molecular Dynamics Simulations”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2024</em>, P01-12, 2024/10/28-31.</li>
    </ul>
  </li>
  <li>2024/10 Poster presentations will be given at Asia &amp; Pacific Bioinformatics Joint Conference 2024.
    <ul>
      <li>Chie Motono, Jun Koseki, <strong>Keisuke Yanagisawa</strong>, Genki Kudo, Ryunosuke Yoshino, Takatsugu Hirokawa, Kenichiro Imai. “Cryptic site detection using machine learning based on mixed-solvent molecular dynamics simulations results”, <em>Asia &amp; Pacific Bioinformatics Joint Conference 2024</em>, 2024/10-22-25.</li>
      <li>Jun Koseki, Chie Motono, <strong>Keisuke Yanagisawa</strong>, Genki Kudo, Ryunosuke Yoshino, Takatsugu Hirokawa, Kenichiro Imai. “Development of the Cryptic Site searching method with Mixed-solvent molecular dynamics and Topological data analyses methods”, <em>Asia &amp; Pacific Bioinformatics Joint Conference 2024</em>, 2024/10-22-25.</li>
    </ul>
  </li>
  <li>2024/08 A paper has been published.
    <ul>
      <li>Jianan Li, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “CycPeptMP: enhancing membrane permeability prediction of cyclic peptides with multi-level molecular features and data augmentation”, <em>Briefings in Bioinformatics</em>, <strong>25</strong>: bbae417, 2024/08. DOI: <a href="https://doi.org/10.1093/bib/bbae417">10.1093/bib/bbae417</a></li>
    </ul>
  </li>
</ul>

<h2 id="2024-january-june">2024 January-June</h2>
<ul>
  <li>2024/06 Poster presentations will be given at the 21st IUPAB and 62nd BSJ joint congress 2024.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Ryunosuke Yoshino, Genki Kudo, Takatsugu Hirokawa. “Quantitative Evaluation of Protein-Compound Substructure Interaction with Inverse Mixed-Solvent Molecular Dynamics Simulation”, <em>21st IUPAB and 62nd BSJ joint congress 2024</em>, 28P-189, 2024/6/28.
        <ul>
          <li><a href="https://speakerdeck.com/keisukeyanagisawa/quantitative-estimation-of-protein-chemical-substructure-interaction-with-inverse-mixed-solvent-molecular-dynamics">Presentation poster</a></li>
        </ul>
      </li>
      <li>Masatake Sugita, Takuya Fujie, Yudai Noso, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “Development and Application of a Protocol for Predicting Membrane Permeability of Cyclic Peptides Based on Molecular Dynamics Simulations”, <em>21st IUPAB and 62nd BSJ joint congress 2024</em>, 26P-210, 2024/06/26.</li>
    </ul>
  </li>
  <li>2024/06 Two oral presentations will be given at IPSJ SIG Bioinformatics and Genomics (SIGBIO).
    <ul>
      <li>Masayoshi Shimizu, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Development of a compound pre-screening method based on spatial arrangement of promising fragment pairs.”, <em>IPSJ SIG Technical Report</em>, <strong>2024-BIO-78</strong>(38): 1-8, 2024/06/21.</li>
      <li>Kaho Akaki, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Acquisition of Bias Information for Protein-Ligand Docking by Mixed-Solvent Molecular Dynamics”, <em>IPSJ SIG Technical Report</em>, <strong>2024-BIO-78</strong>(37): 1-8, 2024/06/21.</li>
    </ul>
  </li>
  <li>2024/06 An oral presentation and a poster presentation will be given at the 24th Annual Meeting of the Protein Science Society of Japan.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Ryunosuke Yoshino, Genki Kudo, Takatsugu Hirokawa. “Quantitative Estimation of Protein-Compound Substructure Interaction with Inverse Mixed-Solvent Molecular Dynamics Simulation”, <em>The 24th Annual Meeting of the Protein Science Society of Japan: [WS-14] Where Bioinformatics and Agrochemistry Meet</em>, 2024/06/13.</li>
      <li>Masatake Sugita, Takuya Fujie, Yudai Noso, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “Development and Application of a Protocol for Predicting Membrane Permeability of Cyclic Peptides Based on Molecular Dynamics Simulations”, <em>The 24th Annual Meeting of the Protein Science Society of Japan: [WS-3] The Present and Future of Peptide Design</em>, 2024/06/11.</li>
      <li>Chie Motono, <strong>Keisuke Yanagisawa</strong>, Genki Kudo, Takatsugu Hirokawa, Kenichiro Imai. “Cryptic site prediction using mixed-solvent molecular dynamics simulation”, <em>The 24th Annual Meeting of the Protein Science Society of Japan</em>, 3P-063, 2024/06/11-13.</li>
    </ul>
  </li>
  <li>2024/04 A paper has been published.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>†, Takuya Fujie, Kazuki Takabatake, Yutaka Akiyama†. “QUBO Problem Formulation of Fragment-Based Protein-Ligand Flexible Docking”, <em>Entropy</em>, <strong>26</strong>: 397, 2024/04. DOI: <a href="https://doi.org/10.3390/e26050397">10.3390/e26050397</a></li>
    </ul>
  </li>
  <li>2024/03 Three oral presentations will be given at IPSJ SIG Bioinformatics and Genomics (SIGBIO).
    <ul>
      <li>Ayako Nunobe, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “フラグメントに基づくバーチャルスクリーニングへの利用などを目指したフラグメント集合の選定”, <em>IPSJ SIG Technical Report</em>, <strong>2024-BIO-77</strong>(31): 1-7, 2024/03/08.</li>
      <li>Yudai Noso, Masatake Sugita, Takuya Fujie, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “分子動力学シミュレーション軌跡データから抽出した位置依存特徴量を活用した環状ペプチドの膜透過性予測”, <em>IPSJ SIG Technical Report</em>, <strong>2024-BIO-77</strong>(16): 1-8, 2024/03/07.</li>
      <li>Jianan Li, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “CycPeptMP: Development of Membrane Permeability Prediction of Cyclic Peptides with Multi-Level Molecular Features and Data Augmentation”, <em>IPSJ SIG Technical Report</em>, <strong>2024-BIO-77</strong>(15): 1-8, 2024/03/07.</li>
      <li>
        <h2 id="2023-july-december">2023 July-December</h2>
      </li>
    </ul>
  </li>
  <li>2023/12 A paper has been published.
    <ul>
      <li>Genki Kudo†, <strong>Keisuke Yanagisawa</strong>†‡, Ryunosuke Yoshino‡, Takatsugu Hirokawa. “AAp-MSMD: Amino Acid Preference Mapping on Protein-Protein Interaction Surfaces Using Mixed-Solvent Molecular Dynamics”, <em>Journal of Chemical Information and Modeling</em>, <strong>63</strong>: 7768-7777, 2023/12. DOI: <a href="https://doi.org/10.1021/acs.jcim.3c01677">10.1021/acs.jcim.3c01677</a></li>
    </ul>
  </li>
  <li>2023/11 A poster presentation will be given at the 61st Annual Meeting of the Biophysical Society of Japan.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Ryunosuke Yoshino, Genki Kudo, Takatsugu Hirokawa. “Quantitative Evaluation of Protein-Chemical Substructure Interaction with Inverse Mixed-Solvent Molecular Dynamics Simulation”, <em>The 61st Annual Meeting of The Biophysical Society of Japan</em>, 2Pos013, 2023/11/14-16.</li>
    </ul>
  </li>
  <li>2023/10 A poster presentation will be given at Chem-Bio Informatics Society(CBI) Annual Meeting 2023.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Ryunosuke Yoshino, Genki Kudo, Takatsugu Hirokawa. “Quantitative Estimation of Protein-Chemical Substructure Interaction with Inverse Mixed-Solvent Molecular Dynamics Simulation”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2023</em>, 2023/10/23-26.</li>
    </ul>
  </li>
  <li>2023/09 An oral presentation will be given at an IIBMP2023 workshop.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>. “Further and Drastic Improvement of AlphaFold is Needed: Insights from Drug Design”, <em>Informatics in Biology, Medicine, and Pharmacology 2023 (IIBMP2023): [WS-2] バイオインフォマティクスの8の問題</em>, 2023/09/05.</li>
    </ul>
  </li>
</ul>

<h2 id="2023-january-june">2023 January-June</h2>
<ul>
  <li>2023/06 Three oral presentations will be given at IPSJ SIG Bioinformatics and Genomics (SIGBIO).
    <ul>
      <li>Jianan Li, <strong>Keisuke Yanagisawa</strong>, Masatake Sugita, Takuya Fujie, Masahito Ohue, Yutaka Akiyama. “CycPeptMPDB: Development of a Comprehensive Database of Membrane Permeability of Cyclic Peptides”, <em>IPSJ SIG Technical Report</em>, <strong>2023-BIO-74</strong>(38): 1-8, 2023/07/01.</li>
      <li>Tomoya Saito, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “フラグメント対の相対位置から検索可能な化合物立体配座データベースの構築”, <em>IPSJ SIG Technical Report</em>, <strong>2023-BIO-74</strong>(37): 1-8, 2023/07/01.</li>
      <li>Ginga Watanabe, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “標的RNAの高次構造予測に基づく低活性ASO候補配列の推測”, <em>IPSJ SIG Technical Report</em>, <strong>2023-BIO-74</strong>(36): 1-8, 2023/07/01.</li>
    </ul>
  </li>
  <li>2023/03 A paper has been published.
    <ul>
      <li>Jianan Li, <strong>Keisuke Yanagisawa</strong>, Masatake Sugita, Takuya Fujie, Masahito Ohue, Yutaka Akiyama. “CycPeptMPDB: A Comprehensive Database of Membrane Permeability of Cyclic Peptides”, <em>Journal of Chemical Information and Modeling</em>, <strong>63</strong>: 2240-2250, 2023/03. DOI: <a href="https://pubs.acs.org/doi/10.1021/acs.jcim.2c01573">10.1021/acs.jcim.2c01573</a></li>
    </ul>
  </li>
  <li>2023/02 I got a JSPS KAKENHI (grant-in-aid for Scientific Research (B)) with two researchers.
    <ul>
      <li>“Designing cyclic peptide with target protein selection based on mixed-solvent molecular dynamics simulation”, JSPS KAKENHI (Grant-in-Aid for Scientific Research (B)) <a href="https://kaken.nii.ac.jp/en/grant/KAKENHI-PROJECT-23H03495/">23H03495</a>, PI, 18 590 000 yen, 2023/04-2027/03.</li>
    </ul>
  </li>
</ul>

<h2 id="2022-july-december">2022 July-December</h2>
<ul>
  <li>2022/10 two oral presentations and a poster presentation had been given at Chem-Bio Informatics Society(CBI) Annual Meeting 2022.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Rikuto Kubota, Yasushi Yoshikawa, Masahito Ohue, Yutaka Akiyama. “REstretto: An efficient protein-ligand docking tool based on a fragment reuse strategy”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2022</em>, O2-1, 2022/10/25.
        <ul>
          <li><a href="https://speakerdeck.com/keisukeyanagisawa/restretto-an-efficient-protein-ligand-docking-tool-based-on-a-fragment-reuse-strategy-cbixue-hui-2022nian-da-hui">Presentation slide</a></li>
        </ul>
      </li>
      <li>Masatake Sugita, Takuya Fujie, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “Lipid composition is critical for accurate membrane permeability prediction of cyclic peptides by molecular dynamics simulations”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2022</em>, O3-2, 2022/10/25.</li>
      <li>Genki Kudo, <strong>Keisuke Yanagisawa</strong>, Ryunosuke Yoshino, Takatsugu Hirokawa. “Amino Acid Preference Mapping on Protein-Protein Interaction Surface using Mixed-Solvent Molecular Dynamics”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2022</em>, P02-04, 2022/10/25-27.</li>
    </ul>
  </li>
  <li>2022/09 A poster presentation will be given at the 60th Annual Meeting of the Biophysical Society of Japan.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Ryunosuke Yoshino, Genki Kudo, Takatsugu Hirokawa. “Inverse Mixed-Solvent Molecular Dynamics for Visualization of Amino Acid Residue Interaction Profile of Molecular Probes”, <em>The 60th Annual Meeting of The Biophysical Society of Japan</em>, 1Pos031, 2022/09/28.</li>
    </ul>
  </li>
  <li>2022/09 A paper has been published.
    <ul>
      <li>Masatake Sugita, Takuya Fujie, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “Lipid composition is critical for accurate membrane permeability prediction of cyclic peptides by molecular dynamics simulations”, <em>Journal of Chemical Information and Modeling</em>, <strong>62</strong>: 4549-4560, 2022/09. DOI: <a href="https://pubs.acs.org/doi/10.1021/acs.jcim.2c00931">10.1021/acs.jcim.2c00931</a></li>
    </ul>
  </li>
  <li>2022/08 A paper has been published.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Rikuto Kubota, Yasushi Yoshikawa, Masahito Ohue, Yutaka Akiyama. “Effective protein-ligand docking strategy via fragment reuse and a proof-of-concept implementation”, <em>ACS Omega</em>, <strong>7</strong>: 30265-30274, 2022/08. DOI: <a href="https://pubs.acs.org/doi/full/10.1021/acsomega.2c03470">10.1021/acsomega.2c03470</a></li>
    </ul>
  </li>
</ul>

<h2 id="2022-january-june">2022 January-June</h2>
<ul>
  <li>2022/06 An oral presentation will be given at IPSJ SIG Bioinformatics and Genomics (SIGBIO).
    <ul>
      <li>Yudai Noso, Masatake Sugita, Takuya Fujie, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “分子動力学シミュレーション軌跡データからの環状ペプチドの膜透過性と相関が高い特徴量の抽出”, <em>IPSJ SIG Technical Report</em>, <strong>2022-BIO-70</strong>(50): 1-8, 2022/06/29.</li>
    </ul>
  </li>
  <li>2022/06 An oral presentation will be given at the 22nd Annual Meeting of the Protein Science Society of Japan (PSSJ2022).
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Ryunosuke Yoshino, Genki Kudo, Takatsugu Hirokawa. “Inverse Mixed-Solvent Molecular Dynamics for Visualization of Residue Interaction Profile of Molecular Probes”, <em>The 22nd Annual Meeting of the Protein Science Society of Japan</em>, O7-12, 2022/06/07.</li>
    </ul>
  </li>
  <li>2022/06 I got a grant for young researcher.
    <ul>
      <li>“Recognition of protein surface where suitable for ligand binding with Inverse-MSMD simulations”, Grant for young researcher, School of Computing, Tokyo Institute of Technology, PI, 500 000 yen, 2022/06-2023/03.</li>
    </ul>
  </li>
  <li>2022/04 A paper has been published.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Ryunosuke Yoshino, Genki Kudo, Takatsugu Hirokawa. “Inverse Mixed-Solvent Molecular Dynamics for Visualization of the Residue Interaction Profile of Molecular Probes”, <em>International Journal of Molecular Sciences</em>, <strong>23</strong>: 4749, 2022/04. DOI: <a href="https://www.mdpi.com/1422-0067/23/9/4749">10.3390/ijms23094749</a></li>
    </ul>
  </li>
  <li>2022/03 Four oral presentation will be given at IPSJ SIG Bioinformatics and Genomics (SIGBIO).
    <ul>
      <li>Yuki Tsushima, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “新たなデータセットによる長距離フラグメントリンキング手法の再評価”, <em>IPSJ SIG Technical Report</em>, <strong>2022-BIO-69</strong>(16): 1-8, 2022/03/11.</li>
      <li>Masaya Inagaki, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “Database of Drug Candidates Represented by 3D Positional Relationships between Fragments”, <em>IPSJ SIG Technical Report</em>, <strong>2022-BIO-69</strong>(15): 1-8, 2022/03/11.</li>
      <li>Mahiro Yamazaki, Kazuki Izawa, Ryo Hirata, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “Development of a Genome-wide Fast Short Nucleotide Sequence Search Method Considering Binding Energy”, <em>IPSJ SIG Technical Report</em>, <strong>2022-BIO-69</strong>(8): 1-8, 2022/03/10.</li>
      <li>Shu Tamano, Kazuki Izawa, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “Proposal of Evaluation Off-target Effects Method in Gapmer ASO”, <em>IPSJ SIG Technical Report</em>, <strong>2022-BIO-69</strong>(7): 1-7, 2022/03/10.</li>
    </ul>
  </li>
  <li>2022/02 A paper has been published.
    <ul>
      <li>Kazuki Takabatake, <strong>Keisuke Yanagisawa</strong>, Yutaka Akiyama. “Solving Generalized Polyomino Puzzles Using the Ising Model”, <em>Entropy</em>, <strong>24</strong>: 354, 2022/02. DOI: <a href="https://doi.org/10.3390/e24030354">10.3390/e24030354</a></li>
    </ul>
  </li>
</ul>

<h2 id="news-archive">News archive</h2>
<p><a href="./news_archive">News archive</a></p>

<p><br /></p>

<h1 id="tools">Tools</h1>

<h2 id="exprorer-2021">EXPRORER (2021)</h2>
<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/exprorer.jpg" alt="EXPRORER" class="align-center middle-img" /></p>

<p>Rational cosolvent set construction with cosolvent MD simulation results</p>

<p><a href="https://github.com/keisuke-yanagisawa/exprorer">https://github.com/keisuke-yanagisawa/exprorer</a></p>

<h2 id="spresso-2017">Spresso (2017)</h2>
<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/spresso.png" alt="Spresso" class="align-center middle-img" /></p>

<p>An ultrafast pre-screening method based on compound decomposition</p>

<p><a href="https://www.bi.cs.titech.ac.jp/spresso">https://www.bi.cs.titech.ac.jp/spresso</a></p>


        
      </section>

      <footer class="page__meta">
        
        


        

      </footer>

      

      
    </div>

    
  </article>

  
  
</div>
    </div>

    
      <div class="search-content">
        <div class="search-content__inner-wrap"><form onsubmit="return googleCustomSearchExecute();" id="cse-search-box-form-id" role="search">
    <label class="sr-only" for="cse-search-input-box-id">
      Enter your search term...
    </label>
    <input type="search" id="cse-search-input-box-id" class="search-input" tabindex="-1" placeholder="Enter your search term..." />
  </form>
  <div id="results" class="results">
    <gcse:searchresults-only></gcse:searchresults-only>
  </div></div>

      </div>
    

    <div id="footer" class="page__footer">
      <footer>
        <!-- start custom footer snippets -->

<!-- end custom footer snippets -->
        <div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    

    

    
      <li><a href="/feed.xml"><i class="fas fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
    
  </ul>
</div>

<div class="page__footer-copyright">&copy; 2025 Keisuke Yanagisawa. Powered by <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>

      </footer>
    </div>

    
  <script src="/assets/js/main.min.js"></script>


<script>
  (function () {
    var cx = '80a5fa516e236c17a';
    var gcse = document.createElement('script');
    gcse.type = 'text/javascript';
    gcse.async = true;
    gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
    var s = document.getElementsByTagName('script')[0];
    s.parentNode.insertBefore(gcse, s);
  })();

  function googleCustomSearchExecute() {
    var input = document.getElementById('cse-search-input-box-id');
    var element = google.search.cse.element.getElement('searchresults-only0');
    if (input.value == '') {
      element.clearAllResults();
    } else {
      element.execute(input.value);
    }
    return false;
  }

  
    $(document).ready(function () {
      $('input#cse-search-input-box-id').on('keyup', function () {
        googleCustomSearchExecute();
      });
    });
  
</script>







  </body>
</html>
