<!doctype html>
<!--
  Minimal Mistakes Jekyll Theme 4.24.0 by Michael Rose
  Copyright 2013-2020 Michael Rose - mademistakes.com | @mmistakes
  Free for personal and commercial use under the MIT license
  https://github.com/mmistakes/minimal-mistakes/blob/master/LICENSE
-->
<html lang="ja" class="no-js">
  <head>
    <meta charset="utf-8">

<!-- begin _includes/seo.html --><title>柳澤 渓甫 | Keisuke Yanagisawa</title>
<meta name="description" content="An assistant professor in Tokyo Tech.">


  <meta name="author" content="Keisuke Yanagisawa">
  


<meta property="og:type" content="website">
<meta property="og:locale" content="ja_JP">
<meta property="og:site_name" content="柳澤 渓甫 | Keisuke Yanagisawa">
<meta property="og:title" content="            柳澤 渓甫      Keisuke Yanagisawa      ">
<meta property="og:url" content="https://keisuke-yanagisawa.github.io/research/">


  <meta property="og:description" content="An assistant professor in Tokyo Tech.">











  

  


<link rel="canonical" href="https://keisuke-yanagisawa.github.io/research/">




<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "Person",
      "name": "Keisuke Yanagisawa",
      "url": "https://keisuke-yanagisawa.github.io/"
    
  }
</script>







<!-- end _includes/seo.html -->



  <link href="/feed.xml" type="application/atom+xml" rel="alternate" title="柳澤 渓甫 | Keisuke Yanagisawa Feed">


<!-- https://t.co/dKP3o1e -->
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="/assets/css/main.css">
<link rel="stylesheet" href="/assets/css/user_define.css">
<link rel="preload" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
<link rel="preload" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
<noscript><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"></noscript>
<noscript><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css"/></noscript>

<!-- for MathJax -->
<script async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js" id="MathJax-script"></script>



    <!-- start custom head snippets -->

<!-- insert favicons. use https://realfavicongenerator.net/ -->

<!-- end custom head snippets -->

  </head>

  <body class="layout--single">
    <nav class="skip-links">
  <ul>
    <li><a href="#site-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
    <li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
    <li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
  </ul>
</nav>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->

    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        
        <a class="site-title" href="https://keisuke-yanagisawa.github.io/en">
          柳澤 渓甫 | Keisuke Yanagisawa
          
        </a>
        <ul class="visible-links">
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/profile">Profile</a>
                </li>
              
            
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/research">Research</a>
                </li>
              
            
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/publications">Publications</a>
                </li>
              
            
              
                <li class="masthead__menu-item" style="color: gray"> | </li>
              
            
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/..">  JP</a>
                </li>
              
            </ul>
        
        <button class="search__toggle" type="button">
          <span class="visually-hidden">Toggle search</span>
          <i class="fas fa-search"></i>
        </button>
        
        <button class="greedy-nav__toggle hidden" type="button">
          <span class="visually-hidden">Toggle menu</span>
          <div class="navicon"></div>
        </button>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>


    <div class="initial-content">
      









<div id="main" role="main">
  
  <div class="sidebar sticky">
  


<div itemscope itemtype="https://schema.org/Person" class="h-card">

  
    <div class="author__avatar">
      <a href="https://keisuke-yanagisawa.github.io/">
        <img src="/assets/img/profile.png" alt="Keisuke Yanagisawa" itemprop="image" class="u-photo">
      </a>
    </div>
  

  <div class="author__content">
    <h3 class="author__name p-name" itemprop="name">
      <a class="u-url" rel="me" href="https://keisuke-yanagisawa.github.io/" itemprop="url">Keisuke Yanagisawa</a>
    </h3>
    <div class="author__bio p-note" itemprop="description">
      <p>
        Assistant Professor, Tokyo Institute of Technology
      </p>
    </div>
  </div>

  <div class="author__urls-wrapper">
    <ul class="author__urls social-icons">
      
        <li itemprop="homeLocation" itemscope itemtype="https://schema.org/Place">
            <i class="fas fa-fw fa-map-marker-alt" aria-hidden="true"></i> <span itemprop="name" class="p-locality">Tokyo, Japan</span>
        </li>
        <br>
      
      
      
        
          
            <li><a href="/assets/CurriculumVitae.pdf" rel="nofollow noopener noreferrer me"><i class="fas fa-fw fa-file-pdf" aria-hidden="true"></i><span class="label">Curriculum Vitae</span></a></li>
          
        
          
            <li><a href="https://orcid.org/0000-0003-0224-0035" rel="nofollow noopener noreferrer me"><i class="fab fa-fw fa-orcid" aria-hidden="true"></i><span class="label">ORCID</span></a></li>
          
        
          
            <li><a href="https://www.researchgate.net/profile/Keisuke-Yanagisawa" rel="nofollow noopener noreferrer me"><i class="fab fa-fw fa-researchgate" aria-hidden="true"></i><span class="label">ResearchGate</span></a></li>
          
        
          
            <li><a href="https://scholar.google.co.jp/citations?user=d-5VVzAAAAAJ" rel="nofollow noopener noreferrer me"><i class="ai fa-fw ai-google-scholar-square" aria-hidden="true"></i><span class="label">Google Scholar</span></a></li>
          
        
          
            <li><a href="http://github.com/keisuke-yanagisawa" rel="nofollow noopener noreferrer me"><i class="fab fa-fw fa-github" aria-hidden="true"></i><span class="label">GitHub</span></a></li>
          
        
          
            <li><a href="https://speakerdeck.com/keisukeyanagisawa" rel="nofollow noopener noreferrer me"><i class="fa-brands fa-speaker-deck" aria-hidden="true"></i><span class="label">Speaker Deck</span></a></li>
          
        
          
            <li><a href="mailto:yanagisawa@c.titech.ac.jp" rel="nofollow noopener noreferrer me"><i class="fas fa-fw fa-envelope-square" aria-hidden="true"></i><span class="label">Email</span></a></li>
          
        
      

      

      <!--
  <li>
    <a href="http://link-to-whatever-social-network.com/user/" itemprop="sameAs" rel="nofollow noopener noreferrer me">
      <i class="fas fa-fw" aria-hidden="true"></i> Custom Social Profile Link
    </a>
  </li>
-->
    </ul>
  </div>
</div>
  
  </div>



  <article class="page h-entry" itemscope itemtype="https://schema.org/CreativeWork">
    
    
    
    

    <div class="page__inner-wrap">
      
        <header>
          
          


        </header>
      

      <section class="page__content e-content" itemprop="text">
        
          <aside class="sidebar__right ">
            
              <nav class="toc">
                <header><h4 class="nav__title"><i class="fas fa-file-alt"></i> On this page</h4></header>
                <ul class="toc__menu"><li><a href="#mixed-solvent-molecular-dynamics-msmd-201904-">Mixed-Solvent Molecular Dynamics (MSMD) (2019/04-)</a><ul><li><a href="#constructing-residue-interaction-profile-of-probes-inverse-msmd">Constructing residue interaction profile of probes (Inverse MSMD)</a></li><li><a href="#msmd-using-amino-acids-as-co-solvents-aap-msmd">MSMD Using Amino Acids as Co-Solvents (AAp-MSMD)</a></li><li><a href="#construction-of-co-solvent-molecule-sets-exprorer">Construction of Co-Solvent Molecule Sets (EXPRORER)</a></li></ul></li><li><a href="#development-of-structure-based-virtual-screening-methods-focused-on-compound-fragments-20144-">Development of Structure-Based Virtual Screening Methods Focused on Compound Fragments (2014/4-)</a><ul><li><a href="#development-of-the-protein-compound-docking-calculation-tool-restretto">Development of the Protein-Compound Docking Calculation Tool REstretto</a></li><li><a href="#docking-calculations-using-quantum-annealers">Docking Calculations Using Quantum Annealers</a></li><li><a href="#development-of-a-fast-compound-pre-screening-method-based-on-compound-decomposition">Development of a Fast Compound Pre-Screening Method Based on Compound Decomposition</a></li></ul></li><li><a href="#prediction-of-drug-clearance-pathways-using-semi-supervised-learning-201304-201508">Prediction of Drug Clearance Pathways Using Semi-Supervised Learning (2013/04-2015/08)</a></li></ul>

              </nav>
            
            
            
          </aside>
        
        <h3 id="mixed-solvent-molecular-dynamics-msmd-201904-">Mixed-Solvent Molecular Dynamics (MSMD) (2019/04-)</h3>

<p>The Mixed-Solvent Molecular Dynamics (MSMD) method uses molecular dynamics (MD) with a protein as the solute and water molecules and probe (co-solvent) molecules. Based on information about where and to what extent co-solvent molecules exist on the protein surface, it is utilized for hotspot detection, binding affinity prediction, and the cryptic binding sites detection induced by structural changes in the protein due to probe molecules.</p>

<h4 id="constructing-residue-interaction-profile-of-probes-inverse-msmd">Constructing residue interaction profile of probes (Inverse MSMD)</h4>

<p>Typically, MSMD maps the probability of the probe around the protein. In contrast, we have developed inverse MSMD to estimate where and what kind of residues of the protein are likely to appear around the probe.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/inverseMSMD.png" alt="inverse MSMD" class="align-center middle-img" /></p>

<p>Additionally, by applying this profile to the protein surface, we are developing a quantitative inverse MSMD method to estimate binding strengths.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/quantitativeinverseMSMD.png" alt="quantitative inverse MSMD" class="align-center middle-img" /></p>

<ul>
  <li><strong>Keisuke Yanagisawa</strong>, Ryunosuke Yoshino, Genki Kudo, Takatsugu Hirokawa. “Inverse Mixed-Solvent Molecular Dynamics for Visualization of the Residue Interaction Profile of Molecular Probes”, <em>International Journal of Molecular Sciences</em>, <strong>23</strong>: 4749, 2022/04. DOI: <a href="https://www.mdpi.com/1422-0067/23/9/4749">10.3390/ijms23094749</a></li>
</ul>

<h4 id="msmd-using-amino-acids-as-co-solvents-aap-msmd">MSMD Using Amino Acids as Co-Solvents (AAp-MSMD)</h4>

<p>By using amino acids as probes, this method predicts protein-protein interaction surfaces, peptide binding sites, and changes in binding affinity due to residue changes.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/AAp-MSMD.jpeg" alt="AAp-MSMD" class="align-center middle-img" /></p>

<ul>
  <li>Genki Kudo†, <strong>Keisuke Yanagisawa</strong>†‡, Ryunosuke Yoshino‡, Takatsugu Hirokawa. “AAp-MSMD: Amino Acid Preference Mapping on Protein-Protein Interaction Surfaces Using Mixed-Solvent Molecular Dynamics”, <em>Journal of Chemical Information and Modeling</em>, <strong>63</strong>: 7768-7777, 2023/12. DOI: <a href="https://doi.org/10.1021/acs.jcim.3c01677">10.1021/acs.jcim.3c01677</a></li>
</ul>

<h4 id="construction-of-co-solvent-molecule-sets-exprorer">Construction of Co-Solvent Molecule Sets (EXPRORER)</h4>

<p>Many studies perform MSMD, but the probes used vary depending on the research target. We have constructed a probe set that can cover frequently occurring substructures in drug molecules by enumerating over 100 frequent structures by dividing drug molecules into substructures and performing MSMD for all of them.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/exprorer.jpg" alt="EXPRORER" class="align-center middle-img" /></p>

<ul>
  <li><strong>Keisuke Yanagisawa</strong>, Yoshitaka Moriwaki, Tohru Terada, Kentaro Shimizu. “EXPRORER: Rational Cosolvent Set Construction Method for Cosolvent Molecular Dynamics Using Large-Scale Computation”, <em>Journal of Chemical Information and Modeling</em>, <strong>61</strong>: 2744-2753, 2021/06. DOI: <a href="https://doi.org/10.1021/acs.jcim.1c00134">10.1021/acs.jcim.1c00134</a></li>
</ul>

<h3 id="development-of-structure-based-virtual-screening-methods-focused-on-compound-fragments-20144-">Development of Structure-Based Virtual Screening Methods Focused on Compound Fragments (2014/4-)</h3>

<p>Structure-based virtual screening (SBVS) uses the 3D structure of target proteins in drug development to select promising compounds from a large set of compounds. This method is widely recognized, but there is room for performance improvement in terms of computation time and accuracy.</p>

<p>We focus on fragments, which are substructures of compounds, knowing that about 28 million compounds can be represented by about 260,000 fragments, aiming to speed up calculations by utilizing fragment commonality among compounds.</p>

<h4 id="development-of-the-protein-compound-docking-calculation-tool-restretto">Development of the Protein-Compound Docking Calculation Tool REstretto</h4>

<p>We are developing REstretto, a protein-compound docking calculation tool that utilizes fragment commonality. This method aims to achieve high-speed docking calculations by formulating them as a QUBO (quadratic unconstrained boolean optimization) problem.</p>

<ul>
  <li><strong>Keisuke Yanagisawa</strong>†, Takuya Fujie, Kazuki Takabatake, Yutaka Akiyama†. “QUBO Problem Formulation of Fragment-Based Protein-Ligand Flexible Docking”, <em>Entropy</em>, <strong>26</strong>: 397, 2024/04. DOI: <a href="https://doi.org/10.3390/e26050397">10.3390/e26050397</a></li>
</ul>

<h4 id="docking-calculations-using-quantum-annealers">Docking Calculations Using Quantum Annealers</h4>
<p>A quantum annealer is a type of quantum computer that is believed to be capable of rapidly solving specific types of combinatorial optimization problems. Therefore, we have formulated docking calculations as QUBO (quadratic unconstrained boolean optimization) problems. While there are many areas for improvement before it becomes practical, it has the potential to achieve a speed performance completely different from conventional methods, making it a subject of our research.</p>

<ul>
  <li><strong>Keisuke Yanagisawa</strong>†, Takuya Fujie, Kazuki Takabatake, Yutaka Akiyama†. “QUBO Problem Formulation of Fragment-Based Protein-Ligand Flexible Docking”, <em>Entropy</em>, <strong>26</strong>: 397, 2024/04. DOI: <a href="https://doi.org/10.3390/e26050397">10.3390/e26050397</a></li>
</ul>

<h4 id="development-of-a-fast-compound-pre-screening-method-based-on-compound-decomposition">Development of a Fast Compound Pre-Screening Method Based on Compound Decomposition</h4>

<p>Evaluating large compound sets (sometimes over hundreds of millions) one by one is computationally difficult, so pre-screening to reduce the compound set is often performed. We are developing Spresso, a pre-screening method that evaluates compounds rapidly based on the 3D structure of the protein.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/spresso.png" alt="Spresso" class="align-center middle-img" /></p>

<ul>
  <li><strong>Keisuke Yanagisawa</strong>, Shunta Komine, Shogo D. Suzuki, Masahito Ohue, Takashi Ishida, Yutaka Akiyama: “Spresso: An ultrafast compound pre-screening method based on compound decomposition”, <em>Bioinformatics</em>, 33: 3836-3843, 2017/12</li>
  <li><strong>Keisuke Yanagisawa</strong>, Shunta Komine, Shogo D. Suzuki, Masahito Ohue, Takashi Ishida, Yutaka Akiyama: “ESPRESSO: An ultrafast compound pre-screening method based on compound decomposition”, <em>The 27th International Conference on Genome Informatics (GIW 2016)</em>, 7 pages, 2016/10</li>
</ul>

<h3 id="prediction-of-drug-clearance-pathways-using-semi-supervised-learning-201304-201508">Prediction of Drug Clearance Pathways Using Semi-Supervised Learning (2013/04-2015/08)</h3>

<p>By calculating molecular weight (MW), partition coefficient (logD), and plasma protein unbound fraction (fup) of drug compounds, we predict the metabolic and excretion pathways (clearance pathways) in the human body.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/clearance.png" alt="Clearance" class="align-center middle-img" /></p>

<p>This prediction problem is characterized by the high cost of labeling (clearance experiments), with a large amount of unlabeled compound structures available. Therefore, semi-supervised learning, which can use unlabeled data, is considered suitable for prediction.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/SL_SSL.png" alt="SL_SSL" class="align-center middle-img" /></p>

<ul>
  <li><strong>Keisuke Yanagisawa</strong>, Takashi Ishida, Yutaka Akiyama: “Drug clearance pathway prediction based on semi-supervised learning”, <em>IPSJ Transactions on Bioinformatics</em>, 8: 21-27, 2015/08</li>
</ul>

        
      </section>

      <footer class="page__meta">
        
        


        

      </footer>

      

      
    </div>

    
  </article>

  
  
</div>
    </div>

    
      <div class="search-content">
        <div class="search-content__inner-wrap"><form onsubmit="return googleCustomSearchExecute();" id="cse-search-box-form-id" role="search">
    <label class="sr-only" for="cse-search-input-box-id">
      Enter your search term...
    </label>
    <input type="search" id="cse-search-input-box-id" class="search-input" tabindex="-1" placeholder="Enter your search term..." />
  </form>
  <div id="results" class="results">
    <gcse:searchresults-only></gcse:searchresults-only>
  </div></div>

      </div>
    

    <div id="footer" class="page__footer">
      <footer>
        <!-- start custom footer snippets -->

<!-- end custom footer snippets -->
        <div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    

    

    
      <li><a href="/feed.xml"><i class="fas fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
    
  </ul>
</div>

<div class="page__footer-copyright">&copy; 2024 Keisuke Yanagisawa. Powered by <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>

      </footer>
    </div>

    
  <script src="/assets/js/main.min.js"></script>


<script>
  (function () {
    var cx = '80a5fa516e236c17a';
    var gcse = document.createElement('script');
    gcse.type = 'text/javascript';
    gcse.async = true;
    gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
    var s = document.getElementsByTagName('script')[0];
    s.parentNode.insertBefore(gcse, s);
  })();

  function googleCustomSearchExecute() {
    var input = document.getElementById('cse-search-input-box-id');
    var element = google.search.cse.element.getElement('searchresults-only0');
    if (input.value == '') {
      element.clearAllResults();
    } else {
      element.execute(input.value);
    }
    return false;
  }

  
    $(document).ready(function () {
      $('input#cse-search-input-box-id').on('keyup', function () {
        googleCustomSearchExecute();
      });
    });
  
</script>







  </body>
</html>
