<!doctype html>
<!--
  Minimal Mistakes Jekyll Theme 4.24.0 by Michael Rose
  Copyright 2013-2020 Michael Rose - mademistakes.com | @mmistakes
  Free for personal and commercial use under the MIT license
  https://github.com/mmistakes/minimal-mistakes/blob/master/LICENSE
-->
<html lang="ja" class="no-js">
  <head>
    <meta charset="utf-8">

<!-- begin _includes/seo.html --><title>柳澤 渓甫 | Keisuke Yanagisawa</title>
<meta name="description" content="An assistant professor in Science Tokyo.">


  <meta name="author" content="Keisuke Yanagisawa">
  


<meta property="og:type" content="website">
<meta property="og:locale" content="ja_JP">
<meta property="og:site_name" content="柳澤 渓甫 | Keisuke Yanagisawa">
<meta property="og:title" content="            柳澤 渓甫      Keisuke Yanagisawa      ">
<meta property="og:url" content="https://keisuke-yanagisawa.github.io/research/">


  <meta property="og:description" content="An assistant professor in Science Tokyo.">











  

  


<link rel="canonical" href="https://keisuke-yanagisawa.github.io/research/">




<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "Person",
      "name": "Keisuke Yanagisawa",
      "url": "https://keisuke-yanagisawa.github.io/"
    
  }
</script>







<!-- end _includes/seo.html -->



  <link href="/feed.xml" type="application/atom+xml" rel="alternate" title="柳澤 渓甫 | Keisuke Yanagisawa Feed">


<!-- https://t.co/dKP3o1e -->
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="/assets/css/main.css">
<link rel="stylesheet" href="/assets/css/user_define.css">
<link rel="preload" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
<link rel="preload" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
<noscript><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"></noscript>
<noscript><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css"/></noscript>

<!-- for MathJax -->
<script async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js" id="MathJax-script"></script>



    <!-- start custom head snippets -->

<!-- insert favicons. use https://realfavicongenerator.net/ -->

<!-- end custom head snippets -->

  </head>

  <body class="layout--single">
    <nav class="skip-links">
  <ul>
    <li><a href="#site-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
    <li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
    <li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
  </ul>
</nav>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->

    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        
        <a class="site-title" href="https://keisuke-yanagisawa.github.io/en">
          柳澤 渓甫 | Keisuke Yanagisawa
          
        </a>
        <ul class="visible-links">
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/profile">Profile</a>
                </li>
              
            
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/research">Research</a>
                </li>
              
            
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/publications">Publications</a>
                </li>
              
            
              
                <li class="masthead__menu-item" style="color: gray"> | </li>
              
            
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/..">  JP</a>
                </li>
              
            </ul>
        
        <button class="search__toggle" type="button">
          <span class="visually-hidden">Toggle search</span>
          <i class="fas fa-search"></i>
        </button>
        
        <button class="greedy-nav__toggle hidden" type="button">
          <span class="visually-hidden">Toggle menu</span>
          <div class="navicon"></div>
        </button>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>


    <div class="initial-content">
      









<div id="main" role="main">
  
  <div class="sidebar sticky">
  


<div itemscope itemtype="https://schema.org/Person" class="h-card">

  
    <div class="author__avatar">
      <a href="https://keisuke-yanagisawa.github.io/">
        <img src="/assets/img/profile.png" alt="Keisuke Yanagisawa" itemprop="image" class="u-photo">
      </a>
    </div>
  

  <div class="author__content">
    <h3 class="author__name p-name" itemprop="name">
      <a class="u-url" rel="me" href="https://keisuke-yanagisawa.github.io/" itemprop="url">Keisuke Yanagisawa</a>
    </h3>
    <div class="author__bio p-note" itemprop="description">
      <p>
        Assistant Professor, Institute of Science Tokyo
      </p>
    </div>
  </div>

  <div class="author__urls-wrapper">
    <ul class="author__urls social-icons">
      
        <li itemprop="homeLocation" itemscope itemtype="https://schema.org/Place">
            <i class="fas fa-fw fa-map-marker-alt" aria-hidden="true"></i> <span itemprop="name" class="p-locality">Tokyo, Japan</span>
        </li>
        <br>
      
      
      
        
          
            <li><a href="/assets/CurriculumVitae.pdf" rel="nofollow noopener noreferrer me"><i class="fas fa-fw fa-file-pdf" aria-hidden="true"></i><span class="label">Curriculum Vitae</span></a></li>
          
        
          
            <li><a href="https://orcid.org/0000-0003-0224-0035" rel="nofollow noopener noreferrer me"><i class="fab fa-fw fa-orcid" aria-hidden="true"></i><span class="label">ORCID</span></a></li>
          
        
          
            <li><a href="https://www.researchgate.net/profile/Keisuke-Yanagisawa" rel="nofollow noopener noreferrer me"><i class="fab fa-fw fa-researchgate" aria-hidden="true"></i><span class="label">ResearchGate</span></a></li>
          
        
          
            <li><a href="https://scholar.google.co.jp/citations?user=d-5VVzAAAAAJ" rel="nofollow noopener noreferrer me"><i class="ai fa-fw ai-google-scholar-square" aria-hidden="true"></i><span class="label">Google Scholar</span></a></li>
          
        
          
            <li><a href="http://github.com/keisuke-yanagisawa" rel="nofollow noopener noreferrer me"><i class="fab fa-fw fa-github" aria-hidden="true"></i><span class="label">GitHub</span></a></li>
          
        
          
            <li><a href="https://speakerdeck.com/keisukeyanagisawa" rel="nofollow noopener noreferrer me"><i class="fa-brands fa-speaker-deck" aria-hidden="true"></i><span class="label">Speaker Deck</span></a></li>
          
        
          
            <li><a href="mailto:yanagisawa@c.titech.ac.jp" rel="nofollow noopener noreferrer me"><i class="fas fa-fw fa-envelope-square" aria-hidden="true"></i><span class="label">Email</span></a></li>
          
        
      

      

      <!--
  <li>
    <a href="http://link-to-whatever-social-network.com/user/" itemprop="sameAs" rel="nofollow noopener noreferrer me">
      <i class="fas fa-fw" aria-hidden="true"></i> Custom Social Profile Link
    </a>
  </li>
-->
    </ul>
  </div>
</div>
  
  </div>



  <article class="page h-entry" itemscope itemtype="https://schema.org/CreativeWork">
    
    
    
    

    <div class="page__inner-wrap">
      
        <header>
          
          


        </header>
      

      <section class="page__content e-content" itemprop="text">
        
          <aside class="sidebar__right ">
            
              <nav class="toc">
                <header><h4 class="nav__title"><i class="fas fa-file-alt"></i> On this page</h4></header>
                <ul class="toc__menu"><li><a href="#docking-calculations-using-quantum-annealer-202304-">Docking Calculations Using Quantum Annealer (2023/04-)</a></li><li><a href="#mixed-solvent-molecular-dynamics-msmd-201904-">Mixed-Solvent Molecular Dynamics (MSMD) (2019/04-)</a><ul><li><a href="#constructing-residue-interaction-profile-of-probes-inverse-msmd">Constructing residue interaction profile of probes (Inverse MSMD)</a></li><li><a href="#msmd-using-amino-acids-as-co-solvents-aap-msmd">MSMD Using Amino Acids as Co-solvents (AAp-MSMD)</a></li><li><a href="#construction-of-co-solvent-molecule-sets-exprorer">Construction of Co-solvent Molecule Sets (EXPRORER)</a></li></ul></li><li><a href="#development-of-structure-based-virtual-screening-methods-focusing-on-compound-fragments-20144-">Development of Structure-Based Virtual Screening Methods Focusing on Compound Fragments (2014/4-)</a><ul><li><a href="#development-of-protein-compound-docking-calculation-tool-restretto">Development of Protein-Compound Docking Calculation Tool REstretto</a></li><li><a href="#development-of-a-fast-compound-pre-screening-methods-based-on-compound-decomposition">Development of a Fast Compound Pre-screening Methods Based on Compound Decomposition</a></li></ul></li><li><a href="#drug-clearance-pathway-prediction-using-semi-supervised-learning-201304-201508">Drug Clearance Pathway Prediction Using Semi-Supervised Learning (2013/04-2015/08)</a></li></ul>

              </nav>
            
            
            
          </aside>
        
        <h2 id="docking-calculations-using-quantum-annealer-202304-">Docking Calculations Using Quantum Annealer (2023/04-)</h2>

<p>A quantum annealer is a type of quantum computer that is expected to rapidly search for local optimal solutions to specific forms of combinatorial optimization problems. We aim to apply this quantum annealer to protein-ligand docking calculations (a method for predicting the binding mode and binding affinity of candidate compounds based on protein three-dimensional structures). To achieve this, <strong>we have converted docking calculations into a format solvable by quantum annealer and formulated them as QUBO (quadratic unconstrained binary optimization) problems</strong>. Specifically, we divide ligand molecules into multiple fragments (substructures) and represent the placement of each fragment with binary variables, thereby expressing the docking search space as QUBO.</p>

<ul>
  <li><strong>Keisuke Yanagisawa</strong>†, Takuya Fujie, Kazuki Takabatake, Yutaka Akiyama†. “QUBO Problem Formulation of Fragment-Based Protein-Ligand Flexible Docking”, <em>Entropy</em>, <strong>26</strong>: 397, 2024/04. DOI: <a href="https://doi.org/10.3390/e26050397">10.3390/e26050397</a></li>
</ul>

<h2 id="mixed-solvent-molecular-dynamics-msmd-201904-">Mixed-Solvent Molecular Dynamics (MSMD) (2019/04-)</h2>

<p>Mixed-solvent molecular dynamics (MSMD) is a molecular dynamics (MD) method that uses proteins as solutes and water molecules along with co-solvent molecules as solvents. Based on information about where and to what extent co-solvent molecules exist on protein surfaces, MSMD is used for hotspot detection and binding affinity evaluation, as well as for discovering cryptic binding sites (hidden binding sites) by utilizing protein structural changes induced by co-solvent molecules.</p>

<p>We have made various proposals regarding this MSMD method.</p>

<h3 id="constructing-residue-interaction-profile-of-probes-inverse-msmd">Constructing residue interaction profile of probes (Inverse MSMD)</h3>

<p>In conventional co-solvent MD, we map where co-solvent molecules are likely to exist on protein surfaces (existence probability distribution). In contrast, <strong>inverse MSMD</strong> performs <strong>estimation of “interaction profiles from the co-solvent perspective,” which determines what types of protein residues are likely to appear around co-solvent molecules</strong>.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/inverseMSMD.png" alt="inverse MSMD" class="align-center middle-img" /></p>

<p>Furthermore, we are currently developing a <strong>quantitative inverse co-solvent MD method</strong> that predicts the strength of actual ligand binding by applying the estimated profiles back to protein surfaces. This is expected to achieve the same results as conventional co-solvent MD without performing MD calculations with target proteins.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/quantitativeinverseMSMD.png" alt="quantitative inverse MSMD" class="align-center middle-img" /></p>

<ul>
  <li><strong>Keisuke Yanagisawa</strong>, Ryunosuke Yoshino, Genki Kudo, Takatsugu Hirokawa. “Inverse Mixed-Solvent Molecular Dynamics for Visualization of the Residue Interaction Profile of Molecular Probes”, <em>International Journal of Molecular Sciences</em>, <strong>23</strong>: 4749, 2022/04. DOI: <a href="https://www.mdpi.com/1422-0067/23/9/4749">10.3390/ijms23094749</a></li>
</ul>

<h3 id="msmd-using-amino-acids-as-co-solvents-aap-msmd">MSMD Using Amino Acids as Co-solvents (AAp-MSMD)</h3>

<p>We developed a method that can analyze the characteristics of protein interactions with other proteins or peptides by using amino acid molecules as co-solvents in MSMD. <strong>AAp-MSMD</strong> performs MSMD using each of the 20 amino acids as small co-solvent molecules. The resulting “preferred location” information for each amino acid residue is useful for predicting protein-protein interactions (PPI) and protein-peptide binding. This is because if locations where specific amino acids preferentially bind on protein surfaces are connected, there is a possibility that this represents a binding site for peptides consisting of several amino acids. It is also possible to qualitatively evaluate the impact of point mutations on binding affinity by comparing how the attachment tendency of surrounding amino acid probe molecules changes when a certain residue on a protein is mutated.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/AAp-MSMD.jpeg" alt="AAp-MSMD" class="align-center middle-img" /></p>

<ul>
  <li>Genki Kudo†, <strong>Keisuke Yanagisawa</strong>†‡, Ryunosuke Yoshino‡, Takatsugu Hirokawa. “AAp-MSMD: Amino Acid Preference Mapping on Protein-Protein Interaction Surfaces Using Mixed-Solvent Molecular Dynamics”, <em>Journal of Chemical Information and Modeling</em>, <strong>63</strong>: 7768-7777, 2023/12. DOI: <a href="https://doi.org/10.1021/acs.jcim.3c01677">10.1021/acs.jcim.3c01677</a></li>
</ul>

<h3 id="construction-of-co-solvent-molecule-sets-exprorer">Construction of Co-solvent Molecule Sets (EXPRORER)</h3>

<p>In MSMD methods, various types of co-solvent molecules have been used depending on research purposes and target proteins, but no clear standards or guidelines exist. Therefore, we attempted to construct a co-solvent molecule set that can comprehensively cover the diverse range of compounds used in drug discovery. We extracted over 100 types of fragments from drug molecules, performed large-scale MSMD simulations, calculated similarities between simulation results, and performed clustering to propose <strong>a co-solvent set that broadly covers “substructures typically contained in drug discovery compounds”</strong>.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/exprorer.jpg" alt="EXPRORER" class="align-center middle-img" /></p>

<ul>
  <li><strong>Keisuke Yanagisawa</strong>, Yoshitaka Moriwaki, Tohru Terada, Kentaro Shimizu. “EXPRORER: Rational Cosolvent Set Construction Method for Cosolvent Molecular Dynamics Using Large-Scale Computation”, <em>Journal of Chemical Information and Modeling</em>, <strong>61</strong>: 2744-2753, 2021/06. DOI: <a href="https://doi.org/10.1021/acs.jcim.1c00134">10.1021/acs.jcim.1c00134</a></li>
</ul>

<h2 id="development-of-structure-based-virtual-screening-methods-focusing-on-compound-fragments-20144-">Development of Structure-Based Virtual Screening Methods Focusing on Compound Fragments (2014/4-)</h2>

<p>Structure-based virtual screening (SBVS) refers to the process of selecting promising compounds from large compound libraries using the three-dimensional structure of target proteins in drug development. While this method has attracted widespread attention, there remains significant room for performance improvement in terms of both computational time and accuracy.</p>

<p>We focus our research on <strong>fragments, which are substructures of compounds</strong>. It is known that <strong>over 28 million compounds can be represented by only about 260,000 fragments</strong>, and we aim to accelerate calculations by utilizing fragment commonality between compounds.</p>

<h3 id="development-of-protein-compound-docking-calculation-tool-restretto">Development of Protein-Compound Docking Calculation Tool REstretto</h3>

<p>To improve the efficiency of docking calculations targeting numerous compounds, we developed a tool called <strong>REstretto</strong> as a docking method based on fragment reuse. REstretto saves information from previously docked fragments and reuses it in docking calculations for other compounds containing the same fragment structures, thereby avoiding computational redundancy. This enables high-speed processing of groups of compounds with similar structures existing in libraries. Experimental evaluation shows that for compound sets of approximately 10,000 compounds, REstretto achieves <strong>computational time and docking accuracy comparable to the widely used AutoDock Vina</strong>. REstretto is publicly available as <a href="https://github.com/akiyamalab/restretto">open source software</a>.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/REstretto.png" alt="REstretto" class="align-center middle-img" /></p>

<ul>
  <li><strong>Keisuke Yanagisawa</strong>, Rikuto Kubota, Yasushi Yoshikawa, Masahito Ohue, Yutaka Akiyama. “Effective protein-ligand docking strategy via fragment reuse and a proof-of-concept implementation”, <em>ACS Omega</em>, <strong>7</strong>: 30265-30274, 2022/08. DOI: <a href="https://pubs.acs.org/doi/full/10.1021/acsomega.2c03470">10.1021/acsomega.2c03470</a></li>
</ul>

<h3 id="development-of-a-fast-compound-pre-screening-methods-based-on-compound-decomposition">Development of a Fast Compound Pre-screening Methods Based on Compound Decomposition</h3>

<p>In virtual screening, which is the initial stage of drug discovery, it is sometimes necessary to evaluate extremely large numbers of compounds, potentially exceeding hundreds of millions. However, performing docking evaluation for each compound individually is computationally unrealistic, so it is common to reduce their number based on compound properties such as molecular weight. However, this filtering can also lead to the exclusion of compounds that bind to target proteins. We developed <strong>Spresso</strong> as a pre-screening method that can rapidly score compounds while using protein three-dimensional structural information.
<img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/spresso.png" alt="Spresso" class="align-center middle-img" /></p>

<ul>
  <li><strong>Keisuke Yanagisawa</strong>, Shunta Komine, Shogo D. Suzuki, Masahito Ohue, Takashi Ishida, Yutaka Akiyama. “Spresso: An ultrafast compound pre-screening method based on compound decomposition”, <em>Bioinformatics</em>, <strong>33</strong>(23): 3836-3843, 2017/12. DOI: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btx178">10.1093/bioinformatics/btx178</a></li>
  <li><strong>Keisuke Yanagisawa</strong>, Shunta Komine, Shogo D. Suzuki, Masahito Ohue, Takashi Ishida, Yutaka Akiyama. “ESPRESSO: An ultrafast compound pre-screening method based on compound decomposition”, <em>The 27th International Conference on Genome Informatics (GIW 2016)</em>, 2016/10/04. (8 pages, Oral Presentation)</li>
</ul>

<h2 id="drug-clearance-pathway-prediction-using-semi-supervised-learning-201304-201508">Drug Clearance Pathway Prediction Using Semi-Supervised Learning (2013/04-2015/08)</h2>

<p>We calculate molecular weight (MW), partition coefficient (logD), plasma protein unbound fraction (fup), and other properties of drug compounds, and use these values to predict which metabolic and excretion pathways (clearance pathways) they will pass through in the human body.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/clearance.png" alt="Clearance" class="align-center middle-img" /></p>

<p>This prediction problem has the characteristic that “the cost of labeling (clearance experiments) is very high, and there are large amounts of unlabeled compound structures available.”
Therefore, semi-supervised learning, which can utilize unlabeled data, is considered more suitable for prediction than commonly used supervised learning.</p>

<p><img src="https://keisuke-yanagisawa.github.io/en/assets/img/research/SL_SSL.png" alt="SL_SSL" class="align-center middle-img" /></p>

<ul>
  <li><strong>Keisuke Yanagisawa</strong>, Takashi Ishida, Yutaka Akiyama: “Drug clearance pathway prediction based on semi-supervised learning”, <em>IPSJ Transactions on Bioinformatics</em>, 8: 21-27, 2015/08
<a href="https://www.jstage.jst.go.jp/article/ipsjtbio/8/0/8_21/_article">[open access]</a></li>
</ul>

        
      </section>

      <footer class="page__meta">
        
        


        

      </footer>

      

      
    </div>

    
  </article>

  
  
</div>
    </div>

    
      <div class="search-content">
        <div class="search-content__inner-wrap"><form onsubmit="return googleCustomSearchExecute();" id="cse-search-box-form-id" role="search">
    <label class="sr-only" for="cse-search-input-box-id">
      Enter your search term...
    </label>
    <input type="search" id="cse-search-input-box-id" class="search-input" tabindex="-1" placeholder="Enter your search term..." />
  </form>
  <div id="results" class="results">
    <gcse:searchresults-only></gcse:searchresults-only>
  </div></div>

      </div>
    

    <div id="footer" class="page__footer">
      <footer>
        <!-- start custom footer snippets -->

<!-- end custom footer snippets -->
        <div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    

    

    
      <li><a href="/feed.xml"><i class="fas fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
    
  </ul>
</div>

<div class="page__footer-copyright">&copy; 2025 Keisuke Yanagisawa. Powered by <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>

      </footer>
    </div>

    
  <script src="/assets/js/main.min.js"></script>


<script>
  (function () {
    var cx = '80a5fa516e236c17a';
    var gcse = document.createElement('script');
    gcse.type = 'text/javascript';
    gcse.async = true;
    gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
    var s = document.getElementsByTagName('script')[0];
    s.parentNode.insertBefore(gcse, s);
  })();

  function googleCustomSearchExecute() {
    var input = document.getElementById('cse-search-input-box-id');
    var element = google.search.cse.element.getElement('searchresults-only0');
    if (input.value == '') {
      element.clearAllResults();
    } else {
      element.execute(input.value);
    }
    return false;
  }

  
    $(document).ready(function () {
      $('input#cse-search-input-box-id').on('keyup', function () {
        googleCustomSearchExecute();
      });
    });
  
</script>







  </body>
</html>
