<!doctype html>
<!--
  Minimal Mistakes Jekyll Theme 4.24.0 by Michael Rose
  Copyright 2013-2020 Michael Rose - mademistakes.com | @mmistakes
  Free for personal and commercial use under the MIT license
  https://github.com/mmistakes/minimal-mistakes/blob/master/LICENSE
-->
<html lang="ja" class="no-js">
  <head>
    <meta charset="utf-8">

<!-- begin _includes/seo.html --><title>柳澤 渓甫 | Keisuke Yanagisawa</title>
<meta name="description" content="An assistant professor in Tokyo Tech.">


  <meta name="author" content="Keisuke Yanagisawa">
  


<meta property="og:type" content="website">
<meta property="og:locale" content="ja_JP">
<meta property="og:site_name" content="柳澤 渓甫 | Keisuke Yanagisawa">
<meta property="og:title" content="            柳澤 渓甫      Keisuke Yanagisawa      ">
<meta property="og:url" content="https://keisuke-yanagisawa.github.io/news_archive/">


  <meta property="og:description" content="An assistant professor in Tokyo Tech.">











  

  


<link rel="canonical" href="https://keisuke-yanagisawa.github.io/news_archive/">




<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "Person",
      "name": "Keisuke Yanagisawa",
      "url": "https://keisuke-yanagisawa.github.io/"
    
  }
</script>







<!-- end _includes/seo.html -->



  <link href="/feed.xml" type="application/atom+xml" rel="alternate" title="柳澤 渓甫 | Keisuke Yanagisawa Feed">


<!-- https://t.co/dKP3o1e -->
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="/assets/css/main.css">
<link rel="stylesheet" href="/assets/css/user_define.css">
<link rel="preload" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5/css/all.min.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
<link rel="preload" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
<noscript><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5/css/all.min.css"></noscript>
<noscript><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css"/></noscript>

<!-- for MathJax -->
<script async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js" id="MathJax-script"></script>



    <!-- start custom head snippets -->

<!-- insert favicons. use https://realfavicongenerator.net/ -->

<!-- end custom head snippets -->

  </head>

  <body class="layout--single">
    <nav class="skip-links">
  <ul>
    <li><a href="#site-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
    <li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
    <li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
  </ul>
</nav>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->

    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        
        <a class="site-title" href="https://keisuke-yanagisawa.github.io/en">
          柳澤 渓甫 | Keisuke Yanagisawa
          
        </a>
        <ul class="visible-links">
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/profile">Profile</a>
                </li>
              
            
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/research">Research</a>
                </li>
              
            
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/publications">Publications</a>
                </li>
              
            
              
                <li class="masthead__menu-item" style="color: gray"> | </li>
              
            
              
                <li class="masthead__menu-item">
                  <a href="https://keisuke-yanagisawa.github.io/en/..">  JP</a>
                </li>
              
            </ul>
        
        <button class="search__toggle" type="button">
          <span class="visually-hidden">Toggle search</span>
          <i class="fas fa-search"></i>
        </button>
        
        <button class="greedy-nav__toggle hidden" type="button">
          <span class="visually-hidden">Toggle menu</span>
          <div class="navicon"></div>
        </button>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>


    <div class="initial-content">
      









<div id="main" role="main">
  
  <div class="sidebar sticky">
  


<div itemscope itemtype="https://schema.org/Person" class="h-card">

  
    <div class="author__avatar">
      <a href="https://keisuke-yanagisawa.github.io/">
        <img src="/assets/img/profile.png" alt="Keisuke Yanagisawa" itemprop="image" class="u-photo">
      </a>
    </div>
  

  <div class="author__content">
    <h3 class="author__name p-name" itemprop="name">
      <a class="u-url" rel="me" href="https://keisuke-yanagisawa.github.io/" itemprop="url">Keisuke Yanagisawa</a>
    </h3>
    <div class="author__bio p-note" itemprop="description">
      <p>
        Assistant Professor, Tokyo Institute of Technology
      </p>
    </div>
  </div>

  <div class="author__urls-wrapper">
    <ul class="author__urls social-icons">
      
        <li itemprop="homeLocation" itemscope itemtype="https://schema.org/Place">
            <i class="fas fa-fw fa-map-marker-alt" aria-hidden="true"></i> <span itemprop="name" class="p-locality">Tokyo, Japan</span>
        </li>
        <br>
      
      
      
        
          
            <li><a href="/assets/CurriculumVitae.pdf" rel="nofollow noopener noreferrer me"><i class="fas fa-fw fa-file-pdf" aria-hidden="true"></i><span class="label">Curriculum Vitae</span></a></li>
          
        
          
            <li><a href="https://orcid.org/0000-0003-0224-0035" rel="nofollow noopener noreferrer me"><i class="fab fa-fw fa-orcid" aria-hidden="true"></i><span class="label">ORCID</span></a></li>
          
        
          
            <li><a href="https://www.researchgate.net/profile/Keisuke-Yanagisawa" rel="nofollow noopener noreferrer me"><i class="fab fa-fw fa-researchgate" aria-hidden="true"></i><span class="label">ResearchGate</span></a></li>
          
        
          
            <li><a href="https://scholar.google.co.jp/citations?user=d-5VVzAAAAAJ" rel="nofollow noopener noreferrer me"><i class="ai fa-fw ai-google-scholar-square" aria-hidden="true"></i><span class="label">Google Scholar</span></a></li>
          
        
          
            <li><a href="http://github.com/keisuke-yanagisawa" rel="nofollow noopener noreferrer me"><i class="fab fa-fw fa-github" aria-hidden="true"></i><span class="label">GitHub</span></a></li>
          
        
          
            <li><a href="mailto:yanagisawa@c.titech.ac.jp" rel="nofollow noopener noreferrer me"><i class="fas fa-fw fa-envelope-square" aria-hidden="true"></i><span class="label">Email</span></a></li>
          
        
      

      

      <!--
  <li>
    <a href="http://link-to-whatever-social-network.com/user/" itemprop="sameAs" rel="nofollow noopener noreferrer me">
      <i class="fas fa-fw" aria-hidden="true"></i> Custom Social Profile Link
    </a>
  </li>
-->
    </ul>
  </div>
</div>
  
  </div>



  <article class="page h-entry" itemscope itemtype="https://schema.org/CreativeWork">
    
    
    
    

    <div class="page__inner-wrap">
      
        <header>
          
          


        </header>
      

      <section class="page__content e-content" itemprop="text">
        
          <aside class="sidebar__right ">
            
              <nav class="toc">
                <header><h4 class="nav__title"><i class="fas fa-file-alt"></i> On this page</h4></header>
                <ul class="toc__menu"><li><a href="#all-news-archive">All news archive</a></li><li><a href="#2021-july-december">2021 July-December</a><ul><li><a href="#2021-january-june">2021 January-June</a></li><li><a href="#2020-july-december">2020 July-December</a></li><li><a href="#2020-january-june">2020 January-June</a></li><li><a href="#2019-july-december">2019 July-December</a></li><li><a href="#2019-april-june">2019 April-June</a></li><li><a href="#fy2018">FY2018</a></li><li><a href="#fy2017">FY2017</a></li><li><a href="#fy2016">FY2016</a></li><li><a href="#fy2015">FY2015</a></li></ul></li></ul>

              </nav>
            
            
            
          </aside>
        
        <h1 id="all-news-archive">All news archive</h1>

<h1 id="2021-july-december">2021 July-December</h1>
<ul>
  <li>2021/11 A paper has been published.
    <ul>
      <li>Jianan Li, <strong>Keisuke Yanagisawa</strong>, Yasushi Yoshikawa, Masahito Ohue, Yutaka Akiyama. “Plasma protein binding prediction focusing on residue-level features and circularity of cyclic peptides by deep learning”, <em>Bioinformatics</em>, <strong>38</strong>: 1110-1117, 2022/02. DOI: <a href="https://doi.org/10.1093/bioinformatics/btab726">10.1093/bioinformatics/btab726</a></li>
    </ul>
  </li>
  <li>2021/10 A paper has been published.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>. “Virtual Screening Methods with a Protein Tertiary Structure for Drug Discovery”, <em>JSBi Bioinformatics Review</em>, <strong>2</strong>: 76-86, 2021/10. DOI: <a href="https://doi.org/10.11234/jsbibr.2021.9">10.11234/jsbibr.2021.9</a> (in Japanese)</li>
    </ul>
  </li>
  <li>2021/09 A paper has been published.
    <ul>
      <li>Kazuki Takabatake, Kazuki Izawa, Motohiro Akikawa, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “Improved Large-Scale Homology Search by Two-step Seed Search Using Multiple Reduced Amino Acid Alphabets”, <em>Genes</em>, <strong>12</strong>: 1455, 2021/09. DOI: <a href="https://doi.org/10.3390/genes12091455">10.3390/genes12091455</a></li>
    </ul>
  </li>
  <li>2021/09 An oral presentation has been accepted in an international conference ICBBS2021.
    <ul>
      <li>Kazuki Takabatake, Kazuki Izawa, Motohiro Akikawa, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “Improved Homology Search for Metagenomic Analysis by Two-Step Seed Search with Reduced Amino Acid Alphabets”, The 10th International Conference on Bioinformatics and Biomedical Science (ICBBS2021), 2021/10/29-31. (Oral Presentation)</li>
    </ul>
  </li>
  <li>2021/07 An oral presentation has been accepted in an international conference PDPTA’21.
    <ul>
      <li>Kazuya Isawa, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “Antisense oligonucleotide activity analysis based on opening and binding energies to targets”, In <em>Proceedings of the 27th International Conference on Parallel and Distributed Processing Techniques and Applications</em> (PDPTA’21), 2021/07/27. (14 pages, Oral Presentation)</li>
    </ul>
  </li>
  <li>2021/07 A paper has been published.
    <ul>
      <li>Masatake Sugita, Satoshi Sugiyama, Takuya Fujie, Yasushi Yoshikawa, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “Large-scale membrane permeability prediction of cyclic peptides crossing a lipid bilayer based on enhanced sampling molecular dynamics simulations”, <em>Journal of Chemical Information and Modeling</em>, <strong>61</strong>: 3681-3695, 2021/7. DOI: <a href="https://doi.org/10.1021/acs.jcim.1c00380">10.1021/acs.jcim.1c00380</a></li>
    </ul>
  </li>
</ul>

<h2 id="2021-january-june">2021 January-June</h2>
<ul>
  <li>2021/06 A paper has been published.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Yoshitaka Moriwaki, Tohru Terada, Kentaro Shimizu. “EXPRORER: Rational Cosolvent Set Construction Method for Cosolvent Molecular Dynamics Using Large-Scale Computation”, <em>Journal of Chemical Information and Modeling</em>, <strong>61</strong>: 2744-2753, 2021/06. DOI: <a href="https://doi.org/10.1021/acs.jcim.1c00134">10.1021/acs.jcim.1c00134</a></li>
    </ul>
  </li>
  <li>2021/03 We won the 2019 Education Award of Excellence from Tokyo Institute of Technology.
    <ul>
      <li>Yoshihiro Miyake, Naoaki Okazaki, Takefumi Kanamori, Tsuyoshi Murata, Shin-ya Nishizaki, Kazuyuki Shudo, Kenji Kise, Masamichi Shimosaka, Masakazu Sekijima, <strong>Keisuke Yanagisawa</strong>, Masahiro Kuze, Mitsuji Sampei, Ichiro Yamanaka, Takehiko Itoh, Toru Takeuchi, Takeo Yamaguchi, Kei Sakaguchi. “Progressive Graduate Minor in Data Science and Artificial Intelligence”</li>
    </ul>
  </li>
</ul>

<h2 id="2020-july-december">2020 July-December</h2>
<ul>
  <li>2020/12 I gave a talk on <a href="https://www2.aeplan.co.jp/mbsj2020/english/index.html">The 43rd Annual Meeting of the Molecular Biology Socienty of Japan</a>.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>. “Rational probe set construction for mixed-solvent molecular dynamics”, <em>The 43rd Annual Meeting of the Molecular Biology Socienty of Japan (MBSJ2020)</em>: [2F-11] forum “State-of-the-art researches of <em>in silico</em> drug discovery”, online, 2020/12/03.</li>
    </ul>
  </li>
  <li>2020/10 I got a grant for young researcher.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, “Computer-aided lead optimization with cosolvent molecular dynamics (CMD)”, Grant for young researcher, School of Computing, Tokyo Institute of Technology, $4 700, 2020/10-2021/03.</li>
    </ul>
  </li>
  <li>2020/07 Our proposal has been accepted the ABCI Grand Challenge Program.
    <ul>
      <li>Yutaka Akiyama, <strong>Keisuke Yanagisawa</strong>, Ohue Masahito, Yasushi Yoshikawa, Masatake Sugita, Takuya Fujie, Kento Aoyama, Satoshi Sugiyama, “Membrane permeability prediction of more than 100 cyclic peptides by REUS simulations and machine learning”, “ABCI Grand Challenge” Program, National Institute of Advanced Industrial Science and Technology (AIST), 2020/08/26-2020/08/27.</li>
    </ul>
  </li>
</ul>

<h2 id="2020-january-june">2020 January-June</h2>
<ul>
  <li>2020/04 I got a JSPS KAKENHI (grant-in-aid for young researchers).
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, “Comprehensive prediction of cryptic binding sites by multi-task deep learning”, JSPS KAKENHI (grant-in-aid for young reseachers), <a href="https://kaken.nii.ac.jp/en/grant/KAKENHI-PROJECT-20K19917/">20K19917</a>, $40 000, 2020/04-2022/03.</li>
    </ul>
  </li>
  <li>2020/04 I joined the School of Computing, Tokyo Institute of Technology as an assistant professor.</li>
  <li>2020/02 I won the Seiichi Tejima Doctoral Dissertation Award from Tokyo Institute of Technology.</li>
</ul>

<h2 id="2019-july-december">2019 July-December</h2>
<ul>
  <li>2019/11 I gave a talk on <a href="https://ahedd2019.ahead-biocomputing.co.jp/">AHeDD2019/IPAB2019 Joint Symposium</a>.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Yoshitaka Moriwaki, Tohru Terada, Kentaro Shimizu. “Systematic construction of the cosolvents sets for cosolvent MD (CMD) with the large-scale simulation”, <em>AHeDD2019/IPAB2019 Joint Symposium</em>, 2019/11.</li>
    </ul>
  </li>
  <li>2019/10 I gave a poster presentation on <a href="https://cbi-society.org/taikai/taikai19/">Chem-Bio Informatics Society(CBI) Annual Meeting 2019</a>.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Yoshitaka Moriwaki, Tohru Terada, Kentaro Shimizu. “Systematic construction of the cosolvents sets for cosolvent MD (CMD) with the large-scale computation”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2019</em>, P1-24, 2019/10.</li>
    </ul>
  </li>
  <li>2019/10 Our proposal has been accepted the Initiative on Promotion of Supercomputing for young or Women Researchers.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, “Construction of the optimal cosolvents sets for cosolvent MD (CMD) with the large-scale computation”, Initiative on Promotion of Supercomputing for Young or Women Researchers, Information Technology Center, The University of Tokyo, $950, 2019/10-2020/03.</li>
    </ul>
  </li>
  <li>2019/09 I gave a poster presentation on <a href="https://www2.aeplan.co.jp/bsj2019/index.html">The 57th Annual Meeting of The Biophysical Society of Japan</a>.
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Yoshitaka Moriwaki, Tohru Terada, Kentaro Shimizu. “Estimation of the probability map (Pmap) similarity of cosolvent MD (CMD) from structural similarities of cosolvents”, <em>The 57th Annual Meeting of The Biophysical Society of Japan</em>, 2019/09.</li>
    </ul>
  </li>
</ul>

<h2 id="2019-april-june">2019 April-June</h2>
<ul>
  <li>2019/04 I joined Bioengineering Information Laboratory, Dept. Biotech., The University of Tokyo as a doctoral fellow (PD).</li>
</ul>

<h2 id="fy2018">FY2018</h2>

<ul>
  <li>2019/03 I got a Ph.D degree in Tokyo Institute of Technology.</li>
  <li>2019/02 A paper was accepted and was published to Molecular Diversity.
    <ul>
      <li>Masahiro Mochizuki, Shogo D. Suzuki, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama. “QEX: Target-specific druglikeness filter enhances ligand-based virtual screening”, <em>Molecular Diversity</em>, 23(1), 11-18, 2019/02. DOI: <a href="https://doi.org/10.1007/s11030-018-9842-3">10.1007/s11030-018-9842-3</a> (open access)</li>
    </ul>
  </li>
  <li>2018/12 BMC Bioinformatics誌に共著論文が採録され、公開されました。
    <ul>
      <li>Takashi Tajimi, Naoki Wakui, <strong>Keisuke Yanagisawa</strong>, Yasushi Yoshikawa, Masahito Ohue, Yutaka Akiyama: “Computational prediction of plasma protein binding of cyclic peptides from small molecule experimental data using sparse modeling techniques”, <em>BMC Bioinformatics</em>, 19(Suppl 19): 527, 2018/12.
<a href="https://doi.org/10.1186/s12859-018-2529-z">[open access]</a></li>
    </ul>
  </li>
  <li>2018/12 <a href="http://datamining-web.it.uts.edu.au/giw2018/">The 29th International Conference on Genome Informatics (GIW 2018)</a>(12/03-12/05, Kunming Univ. Sci. Tech., Yunnan, China)に共著論文が採録されました。
    <ul>
      <li>Takashi Tajimi, Naoki Wakui, <strong>Keisuke Yanagisawa</strong>, Yasushi Yoshikawa, Masahito Ohue, Yutaka Akiyama: “Computational prediction of plasma protein binding of cyclic peptides from small molecule experimental data using sparse modeling techniques”, <em>The 29th International Conference on Genome Informatics (GIW 2018)</em>, 2018/12.</li>
    </ul>
  </li>
  <li>2018/06 Computational Biology and Chemistry誌に論文が採録され、公開されました。
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Shunta Komine, Rikuto Kubota, Masahito Ohue, Yutaka Akiyama: “Optimization of memory use of fragment extension-based protein-ligand docking with an original fast minimum cost flow algorithm”, <em>Computational Biology and Chemistry</em>, 74: 399-406, 2018/06.
<a href="https://doi.org/10.1016/j.compbiolchem.2018.03.013">[open access]</a></li>
    </ul>
  </li>
</ul>

<h2 id="fy2017">FY2017</h2>

<ul>
  <li>2018/02 <a href="https://www.biophysics.org/2018meeting">Biophysical Society 62nd Annual Meeting (BPS18)</a>(02/17-02/21, San Francisco, California, USA)でポスター発表を行いました。
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Shunta Komine, Shogo D. Suzuki, Masahito Ohue, Takashi Ishida, Yutaka Akiyama: “Spresso: An ultrafast compound pre-screening method based on compound fragmentation”, <em>Biophysical Society 62nd Annual Meeting</em>, 2018/02.</li>
    </ul>
  </li>
  <li>2018/01 <a href="http://apbc2018.bio.keio.ac.jp/">The 16th Asia Pacific Bioinformatics Conference (APBC 2018)</a>(01/15-01/17, Keio University, Kanagawa, Japan)にfull paperが採録され、口頭発表を行いました。また、共著full paperも1報採録されました。
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Shunta Komine, Rikuto Kubota, Masahito Ohue, Yutaka Akiyama: “Optimization of memory use of fragment extension-based protein-ligand docking with an original fast minimum cost flow algorithm”, <em>The 16th Asia Pacific Bioinformatics Conference (APBC2018)</em>, 2018/01.</li>
      <li>Takanori Hayashi, Yuri Matsuzaki, <strong>Keisuke Yanagisawa</strong>, Masahito Ohue, Yutaka Akiyama: “MEGADOCK-Web: an integrated database of high-throughput structure-based protein-protein interaction predictions”, <em>The 16th Asia Pacific Bioinformatics Conference (APBC2018)</em>, 2018/01.</li>
    </ul>
  </li>
  <li>2017/12 <a href="http://www.ipab.org/eventschedule/contest/contest4">第4回IT創薬コンテスト「コンピュータで薬のタネを創る4」</a> にてグランプリ（Schrödinger K.K.賞）を獲得しました。
    <ul>
      <li>グループ名「創薬フレンズ」（松山祐輔, 林孝紀, 伊井良太, 種部俊孝, 多治見隆志, 佐藤倫, 池田光, <strong>柳澤渓甫</strong>）: “ストラクチャーベース手法とリガンドベース手法の融合による化合物ヴァーチャルスクリーニング”</li>
    </ul>
  </li>
  <li>2017/08-12 米国 Purdue大学 <a href="http://kiharalab.org/">Kihara Laboratory</a> に滞在しました。</li>
  <li>2017/08 <a href="https://sites.google.com/site/tokyobioinformaticsmeeting/">1st Tokyo Bioinformatics Meeting</a>(08/09, 早大)に参加し、口頭発表を行います。
    <ul>
      <li><strong>柳澤 渓甫</strong>: “Spresso - 高速なタンパク質立体構造ベース創薬を目指して”</li>
    </ul>
  </li>
  <li>2017/06 第50回バイオ情報学(SIGBIO)研究会(06/23-06/25, OIST)で口頭発表を行いました。
    <ul>
      <li><strong>柳澤 渓甫</strong>, 小峰 駿汰, 久保田 陸人, 大上 雅史, 秋山 泰: “フラグメント伸長型化合物ドッキング計算のための重み付きオフラインキャッシュ問題の厳密解アルゴリズム”, <em>情報処理学会研究報告</em>, 2016-BIO-50(38): 1-8, 2017/06</li>
    </ul>
  </li>
  <li>2017/04 日本学術振興会 特別研究員 (DC2) になりました。</li>
</ul>

<h2 id="fy2016">FY2016</h2>

<ul>
  <li>2017/03 Bioinformatics誌に論文が採録され、公開されました。
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Shunta Komine, Shogo D. Suzuki, Masahito Ohue, Takashi Ishida, Yutaka Akiyama: “Spresso: An ultrafast compound pre-screening method based on compound decomposition”, <em>Bioinformatics</em>, 33: 3836-3843, 2017/12.
<a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btx178">[open access]</a></li>
    </ul>
  </li>
  <li>2017/03 第49回バイオ情報学(SIGBIO)研究会(03/23-03/24, JAIST)で口頭発表を行いました。
    <ul>
      <li><strong>柳澤 渓甫</strong>, 大上 雅史, 石田 貴士, 秋山 泰: “標的タンパク質の立体構造を用いたリガンド候補化合物の上限サイズの推定による化合物フィルタリング”, <em>情報処理学会研究報告</em>, 2016-BIO-49(6): 1-7, 2017/03.</li>
    </ul>
  </li>
  <li>2016/10 <a href="http://cbi-society.org/taikai/taikai16/">CBI学会2016年大会</a>(10/25-10/27, タワーホール船堀)でポスター発表を行いました。
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Shunta Komine, Shogo D. Suzuki, Masahito Ohue, Takashi Ishida, Yutaka Akiyama: “ESPRESSO: An ultrafast compound pre-screening method with segmented compounds”, <em>Chem-Bio Informatics Society(CBI) Annual Meeting 2016</em>, 2016/10.</li>
    </ul>
  </li>
  <li>2016/10 <a href="http://admis.fudan.edu.cn/giw2016/">The 27th International Conference on Genome Informatics (GIW 2016)</a>(10/03-10/05, Fudan Univ., Shanghai, China)にfull paperが採録され、口頭発表を行いました。
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Shunta Komine, Shogo D. Suzuki, Masahito Ohue, Takashi Ishida, Yutaka Akiyama: “ESPRESSO: An ultrafast compound pre-screening method based on compound decomposition”, <em>The 27th International Conference on Genome Informatics (GIW 2016)</em>, 2016/10.</li>
    </ul>
  </li>
  <li>2016/09 <a href="http://www.jsbi.org/iibmp2016/">生命医薬情報学連合大会 2016 年大会</a>(09/29-10/01, 東京国際交流館プラザ平成)でlightning talkおよびポスター発表（ポスター番号P65）を行いました。
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Shunta Komine, Shogo D. Suzuki, Masahito Ohue, Takashi Ishida, Yutaka Akiyama: “ESPRESSO: An ultrafast compound pre-screening method based on compound segmentation”, <em>Informatics in Biology, Medicine and Pharmacology 2016</em>, P65, 2016/09.</li>
    </ul>
  </li>
  <li>2016/09 <a href="http://bpwakate.net/summer2016/">第56回生物物理若手の会 夏の学校</a>(09/02-09/05, 支笏湖ユースホステル)に参加しました。</li>
  <li>2016/07 第46回バイオ情報学(SIGBIO)研究会(07/04-07/06, OIST)で口頭発表を行いました。
    <ul>
      <li><strong>柳澤 渓甫</strong>, 小峰 駿汰, 鈴木 翔吾, 大上 雅史, 石田 貴士, 秋山 泰: “フラグメント分割に基づく超高速化合物プレスクリーニング手法 ESPRESSO”, <em>情報処理学会研究報告</em>, 2016-BIO-46(18): 1-7, 2016/07.</li>
    </ul>
  </li>
  <li>2016/04 東京工業大学 情報理工学院 情報工学系 知能情報コースに入学しました。</li>
</ul>

<h2 id="fy2015">FY2015</h2>

<ul>
  <li>2016/03 東京工業大学 大学院情報理工学研究科 計算工学専攻を卒業しました。
    <ul>
      <li>修士論文題目：フラグメント分割に基づく高速な化合物プレドッキング手法の開発</li>
    </ul>
  </li>
  <li>2015/10 <a href="http://biomedinfo.kuicr.kyoto-u.ac.jp/jsbi2015/">生命医薬情報学連合大会 2015 年大会</a>(10/29-10/31, 京大 宇治キャンパス)でlightning talkおよびポスター発表を行いました。
    <ul>
      <li><strong>Keisuke Yanagisawa</strong>, Shunta Komine, Masahito Ohue, Takashi Ishida, Yutaka Akiyama: “Fast pre-filtering for virtual screening based on compound decomposition”, <em>Informatics in Biology, Medicine and Pharmacology 2015</em>, 2015/10.</li>
    </ul>
  </li>
</ul>

        
      </section>

      <footer class="page__meta">
        
        


        

      </footer>

      

      
    </div>

    
  </article>

  
  
</div>
    </div>

    
      <div class="search-content">
        <div class="search-content__inner-wrap"><form onsubmit="return googleCustomSearchExecute();" id="cse-search-box-form-id" role="search">
    <label class="sr-only" for="cse-search-input-box-id">
      Enter your search term...
    </label>
    <input type="search" id="cse-search-input-box-id" class="search-input" tabindex="-1" placeholder="Enter your search term..." />
  </form>
  <div id="results" class="results">
    <gcse:searchresults-only></gcse:searchresults-only>
  </div></div>

      </div>
    

    <div id="footer" class="page__footer">
      <footer>
        <!-- start custom footer snippets -->

<!-- end custom footer snippets -->
        <div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    

    

    
      <li><a href="/feed.xml"><i class="fas fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
    
  </ul>
</div>

<div class="page__footer-copyright">&copy; 2022 Keisuke Yanagisawa. Powered by <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>

      </footer>
    </div>

    
  <script src="/assets/js/main.min.js"></script>


<script>
  (function () {
    var cx = '80a5fa516e236c17a';
    var gcse = document.createElement('script');
    gcse.type = 'text/javascript';
    gcse.async = true;
    gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
    var s = document.getElementsByTagName('script')[0];
    s.parentNode.insertBefore(gcse, s);
  })();

  function googleCustomSearchExecute() {
    var input = document.getElementById('cse-search-input-box-id');
    var element = google.search.cse.element.getElement('searchresults-only0');
    if (input.value == '') {
      element.clearAllResults();
    } else {
      element.execute(input.value);
    }
    return false;
  }

  
    $(document).ready(function () {
      $('input#cse-search-input-box-id').on('keyup', function () {
        googleCustomSearchExecute();
      });
    });
  
</script>







  </body>
</html>
